apps/emqx_dashboard/src/emqx_dashboard_monitor.erl

```diff
@@ -380,6 +385,12 @@ merge_values(disconnected_durable_sessions, M1, M2) ->
 merge_values(Key, M1, M2) ->
     sum_values(Key, M1, M2).
 
+merge_local_values(Key, M1, M2, Gauges) when is_map_key(Key, Gauges) ->
+    %% First argument is assumed to be from a newer timestamp, so we keep the latest.
+    M2#{Key => maps:get(Key, M1, maps:get(Key, M2, 0))};
```

Comment:
call `max_values` ?

Reply:
and max_values needs a fix.
it always return M2 if key is not found in both M1 and M2.

Reply:
it's not the same: we're taking the latest value, not the maximum